api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -35,9 +35,12 @@
 	private String description;
 	
 	private Set<Integer> memberIds;
```

Comment:
Same thing, just update the constructors to create CohortMember objects using the patientIds or Patients